src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,53 @@
+package duke.task;
+
+/**
+ * Implements task objects.
+ * 
+ * @author Audrey Felicio Anwar
+ */
+public class Task {
+    protected String description;
+    protected boolean done;
+
+    public Task(String description, boolean done) {
+        this.description = description;
+        this.done = done;
+    }
+
+    /**
+     * Completes a task.
+     */
+    public void completeTask() {
+        done = true;
+    }
+
+    /**
+     * Describes task.
+     *
+     * @return String that describes task.
+     */
+    @Override
+    public String toString() {
+        String symbol = (this.done ? "[\u2713] " : "[\u2718] ");
+        return symbol + description;
+    }
+
+    /**
+     * Describes task to be saved in hard disk.
+     *
+     * @return String that will be stored on hard disk.
+     */
+    public String saveToHardDisk() {
+        int isDone = done ? 1 : 0;
+        return " | " + isDone + " | " + description;

```

Comment:
Perhaps you can use String.format

Reply:
Thanks for the suggestion. I originally intended not to use `String.format` to improve readability, but I understand this kind of thing is subjective. I will consider using `String.format`.